src/modules/villages/village_service.ts

```diff
@@ -50,4 +52,28 @@ export namespace Village {
 
     return data
   }
+
+  export const findById = async ({ id }: { id: string }): Promise<Entity.FindById> => {
+    const item = await Repository.findById(id)
+
+    if (item === undefined) {
```

Comment:
shortcut use example `if (!item) throw new HttpError(httpStatus.NOT_FOUND, 'village with id ${id} does not exits')`